### ISSUE TITLE
[Bug] Can't automatically calculate correct max value in parallelAxis while datas have multiple categories

### ISSUE THREAD

**Issue by jarvis000000** [apache/echarts#18082]
### Version

5.4.1

### Link to Minimal Reproduction

https://echarts.apache.org/examples/zh/editor.html?c=parallel-aqi

### Steps to Reproduce

1. Delete some datas from dataBJ.
![image](https://user-images.githubusercontent.com/59681983/208620754-e4e92bd8-15d7-4d89-8b93-ec63f292e972.png)

2. The max value and min value in parallelAxis can only calculate by datas in dataBJ.
![image](https://user-images.githubusercontent.com/59681983/208621323-4b399cc4-9304-4fbf-9734-3a7e837f43fa.png)

### Current Behavior

Can't automatically set correct max value in parallelAxis while datas have multiple categories.

### Expected Behavior

The max value and min value in parallelAxis calculate by all datas.

### Environment

```markdown
- OS: Windows 11
- Browser: Chrome 108.0.5359.125
- Framework: Vue@3
```

**Comment by helgasoft** [comment]
Yes, it seems the auto-calculated _min_ and _max_ values of all axes come from the **first series** (dataBJ) only.  
Other series (dataGZ, dataSH) are not taken into account, so min/max values have to be added manually:
```
parallelAxis: [
        {dim: 0, name: schema[0].text, min: ?, max: ?},
        {dim: 1, name: schema[1].text, min: ?, max: ?}, ...
],
```

**Comment by jarvis000000** [comment]
Thanks, I added the min/max values manually. Is there any plan to fix this? I think them have to be auto-aclculated by all series when option whithout them.

**Comment by zym0110** [comment]
这个问题 哪个版本可以修复呢，5以下的版本好像都可以

**Comment by Cahllagerfeld** [comment]
I'm still facing this issue. Did someone find a workaround for it, besides doing a manual calculation for min and max?
